code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -381,3 +381,22 @@
 	R.module.add_module(S, FALSE, TRUE)
 
 	return 1
+
+/obj/item/borg/upgrade/ai
+	name = "b.o.r.i.s. module"

```

Comment:
Oh, thought it was some kind of naming rule against that. Will change in next commit.